packages/core/admin/admin/src/components/FormInputs/Time.tsx

```diff
@@ -32,7 +32,7 @@ const TimeInput = forwardRef<HTMLInputElement, InputProps>(
         }}
         onClear={() => field.onChange(name, undefined)}
         required={required}
-        value={field.value}
+        value={field.value ?? ''}
```

Comment:
ℹ️ Otherwise I have an error coming from the DS trying to do regex string match on null. I saw DatePicker was doing this as well.

Reply:
I don't mind but I can't find the place you're referring to

Reply:
This is where the error comes from in the DS I believe:
https://github.com/strapi/design-system/blob/5062e288af88c4aae8cf36fd52e3600fa36044c3/packages/strapi-design-system/src/TimePicker/TimePicker.tsx#L14

ahh my bad it was StringInput that did this:
https://github.com/strapi/strapi/blob/f237ce8c3df66beb36d02a96297bfdd3c24c838f/packages/core/admin/admin/src/components/FormInputs/String.tsx#L31